library/core/src/num/nonzero.rs

```diff
@@ -895,3 +895,63 @@ macro_rules! nonzero_unsigned_is_power_of_two {
 }
 
 nonzero_unsigned_is_power_of_two! { NonZeroU8 NonZeroU16 NonZeroU32 NonZeroU64 NonZeroU128 NonZeroUsize }
+
+macro_rules! nonzero_constants_signed {
+    ( $( $Ty: ident($Int: ty); )+ ) => {
+        $(
+            impl $Ty {
+                #[unstable(feature = "nonzero_min_max", issue = "89065")]
+                #[doc = concat!("The maximum value for a`", stringify!($Ty), "` is the same as `", stringify!($Int), "`")]
+                /// # Examples
+                #[doc = concat!("assert_eq!(", stringify!($Ty), "::MAX, ", stringify!($Int), "::MAX);")]
+                pub const MAX : $Ty = $Ty::new(<$Int>::MAX).unwrap() ;
```

Comment:
```suggestion
                pub const MAX : $Ty = $Ty::new(<$Int>::MAX).unwrap();
```
Same in the other macro.